pkg/operator/staticpod/controller/common/readyz_conf.go

```diff
@@ -0,0 +1,13 @@
+package common
```

Comment:
this could be moved to https://github.com/openshift/library-go/tree/master/pkg/monitor I think

Reply:
done